pallets/collator-assignment/src/tests/with_core_config.rs

```diff
@@ -0,0 +1,250 @@
+// Copyright (C) Moondance Labs Ltd.
```

Comment:
This file is detached, missing `mod with_core_config` in `src/tests.rs`

Reply:
*fixed*